### PR TITLE
feat: This optimizes and fixes some issues we had with the helm chart

### PR DESCRIPTION
**Pull request by dploeger**
Additionally, includes a cronjob that calls /api/apps every hour to update them

### CODE REVIEW

File: charts/azure-app-exporter/templates/cronjob.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ include "azure-app-exporter.fullname" . }}-refresh-apis
+  labels:
+    {{- include "azure-app-exporter.labels" . | nindent 4 }}
+spec:
+  schedule: "0 * * * *"
```
Comment (mkoertgen) [inline charts/azure-app-exporter/templates/cronjob.yaml:8]: tbd: configurable via helm value?

Comment (dploeger) [reply charts/azure-app-exporter/templates/cronjob.yaml]: Done

**Comment by dploeger** [comment]
@mkoertgen Could this be merged now?

**Comment by mkoertgen** [comment]
Sorry @dploeger for not reacting. This one slipped my attention.

**Pull request closed without merging**